core/utils.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import absolute_import
+
+from pure_pagination import EmptyPage
+from pure_pagination import Paginator
+from django.shortcuts import render
+
+
+def create_child_instance_from_parent(child_cls, parent_instance):
+    """
+    Create a child model instance from a parent instance
+    """
+    parent_cls = parent_instance.__class__
+    field = child_cls._meta.get_ancestor_link(parent_cls).column
+
+    child_instance = child_cls(**{
+        field: parent_instance.pk
+    })
+
+    child_instance.__dict__.update(parent_instance.__dict__)
+    child_instance.save()
+    return child_instance
+
+
+def render_paginate_list(request, object_list, template_name):
+    """
+    Render a paginated list of representatives
+    """
+    pagination_limits = (10, 20, 50, 100)
+    num_by_page = request.GET.get('limit', '30')
```

Comment:
Lol dunno but we should change that indeed it's pretty stupid
On Dec 19, 2015 3:15 PM, "Laurent Peuch" <notifications@github.com> wrote:

> In core/utils.py
> <https://github.com/political-memory/political_memory/pull/44#discussion_r48092733>
> :
>
> > +
> > +    child_instance = child_cls(**{
> > +        field: parent_instance.pk
> > +    })
> > +
> > +    child_instance.__dict__.update(parent_instance.__dict__)
> > +    child_instance.save()
> > +    return child_instance
> > +
> > +
> > +def render_paginate_list(request, object_list, template_name):
> > +    """
> > +    Render a paginated list of representatives
> > +    """
> > +    pagination_limits = (10, 20, 50, 100)
> > +    num_by_page = request.GET.get('limit', '30')
>
> Why the hell uses 30 when 30 isn't in your pagination_limits list?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/political-memory/political_memory/pull/44/files#r48092733>
> .
>